src/components/ImageView/index.js

```diff
@@ -276,14 +285,12 @@ class ImageView extends PureComponent {
                 >
                     <Image
                         source={{uri: this.props.url}}
-                        isAuthTokenRequired={this.props.isAuthTokenRequired}
```

Comment:
It seems `isAuthTokenRequired` this was removed by mistake - this prop should always be forwarded to the image